cmd/server/flag.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	mflag "github.com/itallix/go-metrics/internal/flag"
+)
+
+const (
+	EnvAddress = "ADDRESS"
+)
+
+func parseFlags() *mflag.RunAddress {
```

Comment:
```suggestion
func parseFlags() (*mflag.RunAddress, error) {
```
log.Fatal в случае возникновения ошибки лучше дергать в main() и нигде боле.